tests/blackbox_test.go

```diff
@@ -170,8 +176,16 @@ func outer(t *testing.T, test types.Test, negativeTests bool) {
 	var rootLocation string
 
 	// Setup
-	createFilesFromSlice(t, oemLookasideDir, test.OEMLookasideFiles)
-	createFilesFromSlice(t, systemConfigDir, test.SystemDirFiles)
+	err = createFilesFromSlice(t, oemLookasideDir, test.OEMLookasideFiles)
+	if err != nil {
+		return err
+	}
+	defer os.RemoveAll(oemLookasideDir)
```

Comment:
Having the defer after the error block can potentially leave files/directories existing.